src/Orckestra.Composer.C1.Core/Package/Composite.config.xsl

```diff
@@ -9,4 +10,23 @@
   <xsl:template match="/configuration/Composite.Core.Configuration.Plugins.GlobalSettingsProviderConfiguration/GlobalSettingsProviderPlugins/add/@omitAspNetWebFormsSupport">
     <xsl:attribute name="omitAspNetWebFormsSupport">true</xsl:attribute>
   </xsl:template>
+
+  <xsl:template match="configuration/loggingConfiguration/listeners">
+    <xsl:copy>
+        <xsl:apply-templates select="@* | node()" />
+      <xsl:if test="not(add[@name='AppInsightsListener'])">
+        <add name="AppInsightsListener" listenerDataType="Microsoft.Practices.EnterpriseLibrary.Logging.Configuration.CustomTraceListenerData, Microsoft.Practices.EnterpriseLibrary.Logging, Version=3.0.0.0" traceOutputOptions="None" type="Orckestra.Composer.Website.App_Insights.AppInsightsListener.AppInsightsListener, Orckestra.Composer.Website" initializeData="" formatter="Text Formatter" />
```

Comment:
The namespace for the **AppInsightsListener** class changed, it should probably be updated here as well.